autograd/utils.py

```diff
@@ -6,8 +6,6 @@
 
 
 def train_test_split(X, y, test_size=0.2, random_state=None):
-    np.random.seed(random_state or 1337)
```

Comment:
We will remove this since this util will be used for training actual models with dropout layers